client/rest-high-level/src/main/java/org/elasticsearch/client/security/user/privileges/Role.java

```diff
@@ -294,12 +297,6 @@ public Builder metadata(Map<String, Object> metadata) {
             return this;
         }
 
-        public Builder transientMetadata(Map<String, Object> transientMetadata) {
-            this.transientMetadata =
-                Objects.requireNonNull(transientMetadata, "Transient metadata cannot be null. Pass an empty map instead.");
-            return this;
-        }
-
```

Comment:
Transient metadata cannot be set in the builder, by the client, it is created only by the server.